lib/formatters/verboseFormatter.mjs

```diff
@@ -133,3 +152,28 @@ function ruleLink(rule, metadata) {
 
 	return rule;
 }
+
+/** @param {LintResult[]} results */
+function getFixedRules(results) {
+	/** @type {Record<string, number>} */
+	const rules = {};
+
+	for (const { _postcssResult } of results) {
+		if (!_postcssResult) continue; // CSS syntax error
+
+		const {
+			stylelint: { fixersData },
+		} = _postcssResult;
+
+		for (const ruleName in fixersData) {
+			if (Object.hasOwn(fixersData, ruleName)) {
+				const filteredArray = fixersData[ruleName]?.filter(({ fixed }) => fixed);
+				const count = filteredArray?.length;
+
+				if (count) rules[ruleName] = rules[ruleName] ? rules[ruleName] + count : count;
+			}
+		}
```

Comment:
[suggestion] `for...of` with `Object.entries()` can replace `for...in` with `Object.hasOwn()`. I think the code can be more straightforward.

```suggestion
		for (const [ruleName, fixerData] of Object.entries(fixersData)) {
			const count = fixerData.filter(({ fixed }) => fixed).length;

			if (count) rules.set(ruleName, (rules.get(ruleName) ?? 0) + count);
		}
```